File: class13/python/AP1311-CalculadoraJurosSimples.py
# CalculadoraJurosSimples — Aula 13
# Calcula juros simples: J = C * i * t

capital = float(input("Capital inicial: R$ "))
taxa = float(input("Taxa de juros mensal (%): "))
tempo = int(input("Tempo (meses): "))
juros = capital * (taxa / 100) * tempo
montante = capital + juros
print(f"Juros: R$ {juros:.2f}")
print(f"Montante final: R$ {montante:.2f}")

File: class13/cpp/AP1311-CalculadoraJurosSimples.cpp
#include <iostream>
#include <iomanip>
int main() {
    double c, i; int t;
    std::cout << "Capital inicial: R$ "; std::cin >> c;
    std::cout << "Taxa de juros mensal (%): "; std::cin >> i;
    std::cout << "Tempo (meses): "; std::cin >> t;
    double juros = c * (i/100) * t;
    std::cout << std::fixed << std::setprecision(2);
    std::cout << "Juros: R$ " << juros << std::endl;
    std::cout << "Montante final: R$ " << c+juros << std::endl;
    return 0;
}